src/libs/ReportUtils.ts

```diff
@@ -5676,6 +5676,10 @@ function isDeprecatedGroupDM(report: OnyxEntry<Report>): boolean {
     );
 }
 
+function isRootGroupChat(report: OnyxEntry<Report>): boolean {
```

Comment:
I think we could do with a comment here as `RootGroupChat` isn't immediately obvious to me.

Reply:
Great suggestion!

Reply:
@jjcoffee I added the explain comment.